src/app/components/NotificationModal/NotificationModal.tsx

```diff
@@ -3,23 +3,29 @@ import { FailedVerification } from '../CC/Verification/FailedVerification';
 import { SuccessVerification } from '../CC/Verification/SuccessVerification';
 import useTranslation from 'app/hooks/useTranslation';
 
-export const NotificationModal = ({ handleClose, isOpen, isVerificationSuccess }: NotificationModalProps) => {
+export const NotificationModal = ({
+  customMsg = '',
+  handleClose,
+  isOpen,
+  isVerificationSuccess,
+}: NotificationModalProps) => {
   const t = useTranslation();
 
   return (
     <BaseModal open={isOpen} handleClose={handleClose}>
       <div style={{ textAlign: 'center' }}>
```

Comment:
Is it necessary to have this inline style? Can you move this to some stylesheet?

Reply:
sure I can move it